draft-js-mention-plugin/src/MentionSuggestions/MentionSuggestions.tsx

```diff
@@ -1,14 +1,58 @@
-import React, { Component } from 'react';
+import React, {
+  Component,
+  CSSProperties,
+  FC,
+  KeyboardEvent,
+  ReactElement,
+} from 'react';
 import PropTypes from 'prop-types';
-import { genKey } from 'draft-js';
+import {
+  DraftEditorCommand,
+  DraftHandleValue,
+  EditorState,
+  genKey,
+  SelectionState,
+} from 'draft-js';
 import escapeRegExp from 'lodash/escapeRegExp';
 import Entry from './Entry/Entry';
 import addMention from '../modifiers/addMention';
-import decodeOffsetKey from '../utils/decodeOffsetKey';
+import decodeOffsetKey, { DecodedOffset } from '../utils/decodeOffsetKey';
 import getSearchText from '../utils/getSearchText';
 import defaultEntryComponent from './Entry/DefaultEntryComponent';
+import { MentionData, MentionPluginStore } from '..';
+import { PositionSuggestionsFn } from '../utils/positionSuggestions';
+import { AriaProps } from 'draft-js-plugins-editor/lib';
+import { Theme } from '../theme';
+
+export interface MentionSuggestionCallbacks {
+  keyBindingFn?(e: KeyboardEvent): DraftEditorCommand | string | null;
+  handleKeyCommand: undefined;
+  handleReturn?(e: KeyboardEvent): DraftHandleValue;
```

Comment:
```suggestion
  handleReturn?(event: KeyboardEvent): DraftHandleValue;
```
Here as well. We should prefer full names even in types.